### DISCUSSION TITLE
Why is the NLP task of Pycaret not working?

### DISCUSSION THREAD

**Discussion by shahidulislam-cse** [pycaret/pycaret#3703] [Q&A]
Needs for NLP task automated machine learning library such as **Pycaret**

**Comment by ngupta23** [comment]
It is available in Pycaret < 3.0.0. It was removed from pycaret versions >= 3.0.0

**Reply by shahidulislam-cse** [reply]
ops

**Comment by shahidulislam-cse** [comment]
okay

**Comment by claytonjr** [comment]
Okay, I've read the release notes, and seemingly hit a few of the help requests on github, and searched the greater internet... But there isn't really a meaningful explanation as to why nlp support was removed. Is there a particular reason no one is talking about this out loud? Even better, would someone kindly explain why nlp support was removed?
